IOTATransferRestRequestTemplate.py

```diff
@@ -107,11 +104,11 @@ def create_iota_object(uri, auth_token, seed=None):
 
 # Example of sending a transfer using the sandbox.
 # For more information, see :py:meth:`Iota.send_transfer`.
-# noinspection SpellCheckingInspection
-def create_transaction_dictionary(depth=100, request_tag=None):
+def create_transaction_dictionary(depth=3, request_tag=None):
```

Comment:
Reduced default depth to 3; this is what the light wallets use, I believe.

100 is better for the network, but it takes way too long (: